api/user-story/models/user-story.js

```diff
@@ -7,9 +7,9 @@
 
 module.exports = {
   lifecycles: {
-    async afterUpdate(result) {
-      try {
-        if(result.user_story_status.Status === 'Deployed') {
+    async afterUpdate(result){
+      try{
+          if(result.user_story_status.Status === 'Deployed') {
```

Comment:
here there are 2 additional spaces before the `if`

Reply:
Done